src/workloads/execution/SinusoidalReadWrites.yml

```diff
@@ -0,0 +1,123 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload attempts to create a sinusoidal distribution of reads/writes to test
+  the behavior of execution control when the load on the system changes over time.
+
+  We do this with two actors, one for writes and one for reads and spawn 5000 client
```

Comment:
This comment says 5000 threads but below it looks like the reader and writer each spawn 1000 threads?

Reply:
The 1000 is a default value if no value for "Threads" has been passed in. We pass in 2500 writer threads and 2500 reader threads when creating the write and read actors from this template.

Reply:
Ah I see, do we think those numbers might be too high or is that in line with other workloads? I just want to make sure we're not bottlenecked by the server's ability to handle a large number of connections

Reply:
Changed this to 500 reader/writer threads and lowered the connection pool to 50.